adserver/tasks.py

```diff
@@ -199,5 +206,5 @@ def daily_update_keywords(day=None):
                     keyword=keyword,
                 )
                 KeywordImpression.objects.filter(pk=impression.pk).update(
-                    **{impression_type: values["total"]}
+                    **{impression_type: F(impression_type) + values["total"]}
```

Comment:
I understand the need for this but it's a bit confusing and different from elsewhere. It could probably use a comment. 

I'll just add this discussion for posterity:

> Let’s say you had 10 offers for the keywords “datascience, machine learning” and 5 offers for “datascience, jupyter” or something like that. You want to make sure you have 15 offers for “datascience”